apps/cookbook/src/app/page/side-nav/side-nav.component.ts

```diff
@@ -69,12 +73,16 @@ export class SideNavComponent implements OnInit {
     this.applyComponentFilter('');
   }
 
+  private mapResourcesRoutes() {
+    this.filteredResourceRoutes = this.router.config
```

Comment:
Could we import the routes same way as showcase routes?
```suggestion
    this.filteredResourceRoutes = appRoutes
```